### ISSUE TITLE
Benchmarks for different pretrained models

### ISSUE THREAD

**Issue by McSinyx** [McSinyx/viwikipi#3]
Please comment the benchmarks (F1, accuracy, loss, etc.) here along with instructions to reproduce these results.  Training time would also be helpful.

Models to try out:

- [x] bert-base-multilingual-cased
- [x] xlm-mlm-17-1280

I believe to achieve remarkably better accuracy, we'll need to do some tweaking/addition to the current training data though.

**Comment by McSinyx** [comment]
With Adam ϵ=1e-4 and learning rate=5e-6, after 4 training epochs, the Bert model gives F1 of approximately 0.8.

**Comment by McSinyx** [comment]
XLM log:

```log
2019-11-27 15:10:59.698 | DEBUG    | __main__:<module>:491 - Process rank: -1, device: cuda, n_gpu: 1, distributed training: False
2019-11-27 15:11:53.914 | DEBUG    | __main__:<module>:526 - Training/evaluation parameters Namespace(adam_epsilon=0.0001, cache_dir=None, config=None, cuda=True, data_dir='mrpc', device=device(type='cuda'), eval=True, eval_all_checkpoints=False, eval_while_train=True, gradient_accumulation_steps=1, learning_rate=5e-06, local_rank=-1, log_file='xlm/2019-11-27T15:10:57+00:00.log', logging_steps=None, lower_case=False, max_grad_norm=1.0, max_seq=128, max_steps=-1, model='xlm-mlm-17-1280', model_type='xlm', n_gpu=1, num_train_epochs=3.0, output_dir='xlm', output_mode='classification', overwrite_cache=True, overwrite_output_dir=True, per_gpu_eval_batch_size=8, per_gpu_train_batch_size=8, save_steps=None, seed=42, server_ip=None, server_port=None, task='mrpc', tokenizer=None, train=True, warmup_steps=0, weight_decay=0.0)
2019-11-27 15:25:18.420 | DEBUG    | __main__:evaluate:290 - ***** Eval results checkpoint-1551 *****
2019-11-27 15:25:18.420 | DEBUG    | __main__:evaluate:292 - acc = 0.8576979116075765
2019-11-27 15:25:18.421 | DEBUG    | __main__:evaluate:292 - acc_and_f1 = 0.8389900312484826
2019-11-27 15:25:18.421 | DEBUG    | __main__:evaluate:292 - f1 = 0.8202821508893887
2019-11-27 15:25:18.421 | DEBUG    | __main__:evaluate:292 - loss = 0.3800258531800217
2019-11-27 15:25:18.421 | DEBUG    | __main__:train:189 - average loss = 0.5096873833379347
2019-11-27 15:38:34.338 | DEBUG    | __main__:evaluate:290 - ***** Eval results checkpoint-3102 *****
2019-11-27 15:38:34.339 | DEBUG    | __main__:evaluate:292 - acc = 0.8848955803788247
2019-11-27 15:38:34.339 | DEBUG    | __main__:evaluate:292 - acc_and_f1 = 0.8674319640657576
2019-11-27 15:38:34.339 | DEBUG    | __main__:evaluate:292 - f1 = 0.8499683477526906
2019-11-27 15:38:34.339 | DEBUG    | __main__:evaluate:292 - loss = 0.41474137980283693
2019-11-27 15:38:34.340 | DEBUG    | __main__:train:189 - average loss = 0.40053577265845697
```
